pkg/controller/actions/actions.go

```diff
@@ -11,33 +12,16 @@ import (
 	"github.com/dominodatalab/distributed-compute-operator/pkg/controller/core"
 )
 
-func CreateOrUpdateOwnedResource(ctx *core.Context, owner metav1.Object, controlled client.Object) error {
-	if err := ctrl.SetControllerReference(owner, controlled, ctx.Scheme); err != nil {
-		return err
-	}
+func CreateOwnedResource(ctx *core.Context, owner metav1.Object, controlled client.Object) error {
```

Comment:
@adp312 This seems like it was updated, and could contribute to that Service update bug we saw.